.gitignore

```diff
@@ -26,7 +26,7 @@ platforms/
 plugins/
 plugins/android.json
```

Comment:
Shouldn't `plugins/` be sufficient? Seems like overkill to have `plugins/android.json` and the line after it as well. Not a big deal, but could hurt future maintainability if we assume "it must be in there for a reason" 😌